omnistrate.pro.yaml

```
@@ -34,108 +34,10 @@ x-omnistrate-load-balancer:
-        node-s:
-          <<: *metrics
-        node-sz:
-          <<: *metrics
-        node-mz:
-          <<: *metrics
-        cluster-sz:
-          <<: *metrics
-        cluster-mz:
-          <<: *metrics
-
+  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)